policies/project-director-election-process.md

```diff
@@ -0,0 +1,60 @@
+# Project Director Election Process
+
+## Aims
+
+The purpose of this policy is to outline a clear and efficient process for electing new Project Directors to represent the Rust project on the board of the Rust Foundation. The process aims to achieve the following goals:
+
+* Select candidates who fulfill the outlined candidate criteria, focusing on shared power and new leadership rather than solely prioritizing the best candidate.
+    * Strive for tolerance and inclusivity, aiming for the overlap of tolerance rather than the overlap of preference.
+* Conduct the selection process in a time-efficient manner.
+* Incorporate input from a wide range of stakeholders across the entire Rust project.
+* Ensure a diverse pool of qualified candidates for selection.
+* Create a psychologically safe environment that values minority perspectives and encourages positive feedback, encouragement, and tolerance rather than preference or criticism.
+* Ensure participants feel positive about their interactions and the execution of the process.
+
+## Description
+
+### Setup
+
+* The council selects a facilitator by consent:
+    * Any team member can suggest a facilitator or volunteer themselves.
+    * The facilitator must not be a nominee in the election and must be someone everyone trusts to set aside personal preferences during the election process.
+    * **Aims of the facilitator role**
+        * Clearly documenting and communicating the process and announcing and supporting its execution.
+        * Tracking, reviewing, and documenting the effectiveness of the process.
+* Review or Define Role Description ([example](https://github.com/rust-lang/rfcs/pull/3392#issuecomment-1505697944))
```

Comment:
I think it makes sense to emphasize the "Review" aspect of this point. I suspect we want the role to be well defined before beginning the election process. We might want to revise the role over time, but I feel like this revision process should happen separate from the election process and we should expect relatively small changes to the role each time around. Saying "define" suggests to me that we are throwing out the old description (which describes the job the current PDs are already doing) and starting from scratch on a description (which would likely end up being close to what we already had).

I definitely think it makes sense to review the role description though, like "we are looking for people who would be successful at doing the role as already defined."

Reply:
The way that I intended this was for people to read "define" when there isn't already a role description and "review" when there is one. 

Do you have alternative wording you feel would be more clear? If not I can try my hand at something.

Reply:
I suppose since there already is a description, we can just remove "define"?